src/pages/billing/Project.js

```diff
@@ -568,6 +615,7 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
       ])
     ]),
     [spendReportKey]: div({ style: { display: 'grid', rowGap: '0.5rem' } }, [
+      !!errorMessage && h(ErrorAlert, { error, errorMessage }),
```

Comment:
It looks like this is the only place `error` is used, so `ErrorAlert` could take one prop (`errorMessage`) and parse it itself.